docs/administrator/verify-artifacts.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Verify Artifacts
+---
+
+## verify images
+
+Karmada has introduced cosign to verify the released images since version v1.7. The specific operation is as follows：
+
+### Prerequisites
+
+You need to install the following tools:
+
+- `cosign` ([Installation Guide](https://docs.sigstore.dev/cosign/installation/))
+- `curl` (usually provided by your OS)
+- `jq` ([download jq](https://stedlan.github.io/jq/download/))
+
+### Verify image signature
+
+#### Verify image with cosign CLI
+
+Karmada hosts the image to `dockerhub`, and introduces the `cosign` verification tool after v1.7. For a list of published mirrors, see [karmada mirrors](https://hub.docker.com/u/karmada).
```

Comment:
```suggestion
Karmada introduced the `cosign` verification tool since release 1.7. For a list of published mirrors, see [karmada mirrors](https://hub.docker.com/u/karmada).
```